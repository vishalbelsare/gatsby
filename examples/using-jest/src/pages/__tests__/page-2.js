/**
 * @jest-environment jsdom
 */

import * as React from "react"
import { render } from "@testing-library/react"

import PageTwo from "../page-2"

describe(`PageTwo`, () => {
  it("renders correctly", () => {
    const data = {
      site: {
        siteMetadata: {
          author: "Your name",
        },
      },
    }

    const { container } = render(<PageTwo data={data} />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
