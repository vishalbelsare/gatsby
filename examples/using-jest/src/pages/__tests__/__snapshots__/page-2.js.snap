// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTwo renders correctly 1`] = `
<main>
  <h1>
    Page 2
  </h1>
  <p>
    Your name
  </p>
</main>
`;
